spec/features/journeys/basic_registration_journey_spec.rb

```diff
@@ -3,12 +3,22 @@
 RSpec.feature "Happy journeys", type: :feature do
   include Helpers::JourneyHelper
   include Helpers::JourneyAssertionHelper
+  include Helpers::JourneyStepHelper
 
   include_context "retrieve latest application data"
   include_context "stub course ecf to identifier mappings"
   include_context "Enable Get An Identity integration"
 
-  scenario "registration journey" do
+  context "when JavaScript is enabled", :js do
+    scenario("registration journey (with JS)") { basic_registration_journey(js: true) }
+  end
+
+  context "when JavaScript is disabled", :no_js do
+    include_context "use rack_test driver"
+    scenario("registration journey (without JS)") { basic_registration_journey(js: false) }
+  end
```

Comment:
Use RSpec's tags to be able to easily target the ones we want. I'll probably disable `:js` ones locally most of the time as they take hundreds of times longer than the `:no_js` ones